Nearby crash when LatLng is null

**Summary:** 

I am able to get this crash on some first runs. Then it gets solved.

**Steps to reproduce:** 

Not very stable, sometimes, it happens. Ie, once I turned off wireless connection and it happened.
**Add System logs:**
```
                                 Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'double android.location.Location.getLatitude()' on a null object reference
                                                                             at fr.free.nrw.commons.location.LatLng.from(LatLng.java:45)
                                                                             at fr.free.nrw.commons.location.LocationServiceManager.getLKL(LocationServiceManager.java:109)
                                                                             at fr.free.nrw.commons.nearby.NearbyActivity.onRequestPermissionsResult(NearbyActivity.java:224)
                                                                             at android.app.Activity.dispatchRequestPermissionsResult(Activity.java:6553)
                                                                             at android.app.Activity.dispatchActivityResult(Activity.java:6432)
```

--- a/app/src/main/java/fr/free/nrw/commons/location/LocationServiceManager.java
+++ b/app/src/main/java/fr/free/nrw/commons/location/LocationServiceManager.java
@@ -106,6 +106,9 @@ public class LocationServiceManager implements LocationListener {
             if (lastKL == null) {
                 lastKL = locationManager.getLastKnownLocation(LocationManager.NETWORK_PROVIDER);
             }
+            if (null == lastKL) {
+                return null;
+            }
             return LatLng.from(lastKL);
         } else {
             return null;

--- a/app/src/main/java/fr/free/nrw/commons/location/LocationServiceManager.java
+++ b/app/src/main/java/fr/free/nrw/commons/location/LocationServiceManager.java
@@ -106,7 +106,7 @@ public class LocationServiceManager implements LocationListener {
             if (lastKL == null) {
                 lastKL = locationManager.getLastKnownLocation(LocationManager.NETWORK_PROVIDER);
             }
-            if (null == lastKL) {
+            if (lastKL == null) {
                 return null;
             }
             return LatLng.from(lastKL);

